src/current/v24.1/disaster-recovery-overview.md

```diff
@@ -0,0 +1,172 @@
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Automatic</td>
+    <td>Automatic</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x<br>Networking between availability zones</td>
+    <td>1x<br>Networking between datacenters</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Minimum regions</b>
+    </td>
+    <td>1</td>
+    <td>3</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Fault tolerance</b>
+    </td>
+    <td>Zero RPO node, availability zone failures</td>
+    <td>Zero RPO node, availability zone failures</td>
+  </tr>
+
+</table>
+
+For detail on designing your cluster topology for high availability with replication, refer to the [Disaster Recovery Planning]({% link {{ page.version.version }}/disaster-recovery-planning.md %}#hardware-failure) page.
+
+### Choose a disaster recovery strategy
+
+CockroachDB is designed to recover automatically, however building [backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) or [physical cluster replication]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) into your disaster recovery planning is an important part of a resilient deployment.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Point in time backup & restore</th>
+    <th>Physical cluster replication (async)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>>=5 minutes</td>
+    <td>10s of seconds</a></td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Minutes to hours, depending on data size and number of nodes</td>
+    <td>Seconds to minutes, depending on cluster size</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>No impact</td>
+    <td>No impact</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Manual restore</td>
+    <td>Manual failover</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x + S3-like storage & networking</td>
+    <td>2x + networking between datacenters</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Minimum regions</b>
+    </td>
+    <td>1</td>
+    <td>2</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Fault tolerance</b>
+    </td>
+    <td>Not applicable</td>
+    <td>Zero RPO node, availability zone region failure with loss up to RPO</td>
```

Comment:
nit: i think a comma is needed after 'availability zone'?

Reply:
Oh yeah, looks like it! Added the comma!